--- hdf5-1.8.12.orig/test/Makefile.in   2013-11-21 15:14:03.000000000 +0000
+++ hdf5-1.8.12/test/Makefile.in    2017-07-14 10:47:16.000000000 +0100
@@ -866,13 +866,13 @@
 # These tests (fheap, btree2) are under development and are not used by
 # the library yet. Move them to the end so that their failure do not block
 # other current library code tests.
-TEST_PROG = testhdf5 lheap ohdr stab gheap cache cache_api \
+TEST_PROG = testhdf5 lheap ohdr stab gheap cache_api \
            pool accum hyperslab istore bittests dt_arith \
            dtypes dsets cmpd_dset filter_fail extend external efc objcopy links unlink \
            big mtime fillval mount flush1 flush2 app_ref enum \
            set_extent ttsafe \
            getname vfd ntypes dangle dtransform reserved cross_read \
-           freespace mf btree2 fheap file_image unregister
+           freespace mf btree2 file_image unregister


 # These programs generate test files for the tests.  They don't need to be